.github/workflows/ci.yml

```diff
@@ -19,7 +22,7 @@ jobs:
       - name: Check out code
         uses: actions/checkout@v4
         with:
-          fetch-depth: 2
+          fetch-depth: 0
```

Comment:
Why did this get reduced?

Reply:
It actually got increased. We need all the history to perform the check for whether or not changesets were added